### PR TITLE
 Test PUT and GET

### PR DESCRIPTION
**Pull request by jeffrey20122021**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,15 +23,19 @@
...
     "eslint-plugin-import": "^2.26.0",
     "http-status-codes": "^2.2.0",
+    "karma-chai-subset": "^2.0.1",
     "mocha": "^10.0.0",
     "mochawesome": "^7.1.3"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^0.27.2",
+    "md5": "^2.3.0"
```
Comment (holgiosalos) [inline package.json]: md5 dependency should be a dev dependency. To fix it, run: `npm i -D md5`

File: test/GithubApi.Repositories.test.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    });
+    it('Via OAuth2 Tokens by parameter', async () => {
+      const response = await axios.get(
+        'https://api.github.com/users/aperdomob',
+        { access_token: process.env.ACCESS_TOKEN }
+      );
+      expect(response.status).to.equal(StatusCodes.OK);
+    });
+  });
+});
```
Comment (holgiosalos) [inline test/GithubApi.Repositories.test.js]: This is not required for this file.

Comment (jeffrey20122021) [reply test/GithubApi.Repositories.test.js]: I delete this part and did the test again in github and do not works. In the last commit all test pass.

File: package.json (hunk 1 above)
Comment (jeffrey20122021) [reply package.json]: i did it. Thanks

File: test/GithubApi.Repositories.test.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+const { StatusCodes } = require('http-status-codes');
```
Comment (holgiosalos) [inline test/GithubApi.Repositories.test.js]: remove this import statement since it is not used, it makes the linter fail

**Review by holgiosalos — COMMENTED** [review]
run the lint fix command:
`npm run lint -- --fix`

File: test/GithubApi.Repositories.test.js (hunk 3 above)
Comment (jeffrey20122021) [reply test/GithubApi.Repositories.test.js]: Done

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**